ietf-microwave-radio-link.yang

```diff
@@ -121,6 +126,45 @@ module ietf-microwave-radio-link {
        for configuration or status.";
   }
 
+  grouping header-compression-profile {
+  leaf header-compression-profile-name {
+    type string;
+    description
+      "none";
+  }
+  description
+    "Header compression profile: can be a list of protocol header to be compressed
+     or a header size in which all included (also partially) headers get compressed.";
+    choice header-compression {
+      case protocol-based {
+        leaf-list compressed-protocol-layer-list {
```

Comment:
No need to add the '-list' suffix

```suggestion
        leaf-list compressed-protocol-layer {
```

Adding a top-level container 'protocol-based' may also help checking which modes are supporting protocol-based header compression and which modes are not